includes/definitions/discovery/riverbed.yaml

```diff
@@ -0,0 +1,44 @@
+mib: STEELHEAD-MIB:RBT-MIB
+modules:
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: systemTemperature
+                    value: systemTemperature
+                    num_oid: .1.3.6.1.4.1.17163.1.1.2.9.
+                    snmp_flags: '-OQUs'
+                    index: 0
+                    descr: 'System Temperature'
+                    high_limit: 65
+        state:
+            data:
+                -
+                    oid: systemHealth
+                    value: systemHealth
+                    num_oid: .1.3.6.1.4.1.17163.1.1.2.7.
+                    descr: 'System Health'
+                    index: 0
+                    state_name: systemHealth
+                    states:
+                        - { descr: healthy, graph: 1, value: 10000, generic: 0 }
+                        - { descr: degraded, graph: 1, value: 30000, generic: 1 }
+                        - { descr: admissionControl, graph: 1, value: 31000, generic: 1 }
+                        - { descr: critical, graph: 1, value: 50000, generic: 2 }
+                -
+                    oid: optServiceStatus
+                    value: optServiceStatus
+                    num_oid: .1.3.6.1.4.1.17163.1.1.2.8.
+                    descr: 'Optimization Service Status'
+                    index: 0
+                    state_name: optServiceStatus
+                    states:
+                       - { descr: none, graph: 1, value: 0, generic: 3 }
```

Comment:
This has a missing space which is breaking the yaml: http://www.yamllint.com/

Reply:
Thank you! Fixed.